acme/authority.go

```diff
@@ -43,11 +44,35 @@ type Authority struct {
 	signAuth SignAuthority
 }
 
+var (
+	accountTable           = []byte("acme_accounts")
+	accountByKeyIDTable    = []byte("acme_keyID_accountID_index")
+	authzTable             = []byte("acme_authzs")
+	challengeTable         = []byte("acme_challenges")
+	nonceTable             = []byte("nonces")
```

Comment:
Why not acme_nonces?

Reply:
I figured we might use `nonces` in other places as well ...

Reply:
ok